### PR TITLE
Blazor Fix Missing EventArgs.Type

### LINKED ISSUE #32397: TouchEventArgs.Type is always null
Also true of MouseEventArgs.Type. It does not look like these translated from browser's UIEventArgs.

### LINKED ISSUE #38782: Blazor - MouseEventArgs.Type property is always null in .NET 6.0
Hi,
In my Blazor .NET 5.0 project, I used one delegate function for several mouse events, e.g .: 

@code {
async Task OnMouseEvent(MouseEventArgs e)    {
        switch (e.Type)
        {
            case "click": 
            //...
            break;
            case "dblclick": 
            //...
            break;
            case "mousemove": 
            //...
            break;
//...

After upgrading the project to .NET 6.0, the Type property of MouseEventArgs parameter is always null.
Is it intentional or is it a bug?

Thanks a lot

### PR DESCRIPTION
**Pull request by TanayParikh**
Fixes: https://github.com/dotnet/aspnetcore/issues/38782
Fixes: https://github.com/dotnet/aspnetcore/issues/32397

### CODE REVIEW

File: src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor [hunk 1]
```diff
@@ -43,49 +43,74 @@
     void OnMouseOver(MouseEventArgs e)
     {
         DumpEvent(e);
-        message += "onmouseover,";
+
+        if (e.Type == "mouseover")
```
Comment (pranavkm) [inline src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor:47]: For these ones, couldn't we look at the dumped event? I imagine that's what these tests already do?

**Review by pranavkm — APPROVED** [review]
The source code change looks fine, although I'd like to figure out if we could write the test without having to special case this one property.

File: src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor (hunk 1 above)
Comment (TanayParikh) [reply src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor:47]: Dumped events are dumped to the command line. I guess we can dump the `Type` specifically to another `message` like var and confirm it on the C# side. Updating...

Comment (TanayParikh) [reply src/Components/test/testassets/BasicTestApp/MouseEventComponent.razor]: Hows this: https://github.com/dotnet/aspnetcore/pull/38792/commits/d40bae4deb7425936e70da135df13e5c2ed0878e

File: src/Components/test/E2ETest/Tests/EventTest.cs [hunk 2]
```diff
@@ -174,10 +174,10 @@ public void DragDrop_CanTrigger()
 
         actions.Perform();
         // drop doesn't seem to trigger in Selenium. But it's sufficient to determine "any" drag event works
-        Browser.Equal("ondragstart,", () => output.Text);
+        Browser.Equal("dragstart,", () => output.Text);
     }
 
-    [Fact(Skip = "https://github.com/dotnet/aspnetcore/issues/32373")]
+    [Fact]
```
Comment (TanayParikh) [inline src/Components/test/E2ETest/Tests/EventTest.cs:180]: This may be resolved given the below change (was missing a comma), trying it out on the CI.

**Review by pranavkm — APPROVED** [review]
:shipit:

**Pull request merged**